src/dotnet/CarbonAware.WebApi.Tests/unitTests/CarbonAwareControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+namespace CarbonAware.WepApi.UnitTests;
+
+using System.Collections.Generic;
+using CarbonAware.Model;
+using CarbonAware.WebApi.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+/// <summary>
+/// Testing Web API Controller
+/// </summary>
+[TestFixture]
+public class CarbonAwareControllerTests : TestsBase
+{   
+    /// <summary>
+    /// Tests that successfull call to plugin results in action with 200 status
+    /// </summary>
+    [Test]
+    public async Task SuccessfulCallReturnsOk()
+    {
+        string location = "Sydney";
+        var data = new List<EmissionsData>()
+        {
+            new EmissionsData()
+            {
+                Location = location,
+                Rating = 0.9,
+                Time = DateTime.Now
+            }
+        };
+        var controller = new CarbonAwareController(this.MockLogger.Object, CreatePluginWithData(data).Object);
+        
+        IActionResult ar1 = await controller.GetEmissionsDataForLocationByTime(location);
+        IActionResult ar2 = await controller.GetBestEmissionsDataForLocationsByTime(new string[] { location });
+        IActionResult ar3 = await controller.GetEmissionsDataForLocationsByTime(new string[] { location });
+
+        TestHelpers.AssertStatusCode(ar1, 200);
```

Comment:
This should be using the built-in dotnet status code lookup tool -> https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-6.0